CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+Hi! 👋 We’re excited that you’re interested in contributing!
+
+## Core Guidelines
+
+[Read the guidelines on the MDX website][contributing]
+
+## Packages Releasing Difference
+
+This project is a [lerna][] monorepo, so packages releasing is controlled by [lerna][].
+
+1. Make sure you have both GitHub repository and npm write permissions at the same time.
+2. Export an environment named `GH_TOKEN` for `lerna publish`.
```

Comment:
People shouldn’t directly use `lerna publish`, right? If it shouldn’t be used, I suggest removing this bullet, as the next one already explains how to pass a GH token?

Reply:
@wooorm I meant to talk about exporting the environment globally inside something like `.bash_profile`.

Reply:
```suggestion
2. You need a GitHub token with a `public_repo` scope as `GH_TOKEN` in the
   environment to publish
```

Maybe like this?